client/ayon_core/pipeline/load/plugins.py

```diff
@@ -251,15 +252,50 @@ class ProductLoaderPlugin(LoaderPlugin):
     """
 
 
+class PrePostLoaderHookPlugin:
+    """Plugin that should be run before or post specific Loader in 'loaders'
+
+    Should be used as non-invasive method to enrich core loading process.
+    Any studio might want to modify loaded data before or after
+    they are loaded without need to override existing core plugins.
+    """
+    @classmethod
+    def is_compatible(cls, Loader: LoaderPlugin) -> bool:
+        pass
```

Comment:
I'd argue that this should default to `True` so that by default it applies to all.

Reply:
Or it should be an abstract method because currently it does not return bool by default.

Reply:
For what it's worth - as soon as I started implementing as a test it would have made my life easier if this were just `return True` by default - since I can build and test the implementation with less. (I missed adding the `is_compatible` initially).

But I suppose abstract method would similarly provide me with hints in that area to implement it. 👍